python-main.js

```diff
@@ -1108,15 +1108,17 @@ function web_editor(config) {
             errorMessage = "update-req";
         } else if (err.message === "Unable to claim interface.") {
             errorMessage = "clear-connect";
+        } else if (err.name === "device-disconnected") {
+            errorMessage = "device-disconnected";
         } else {
             errorMessage = "reconnect-microbit";
         }
 
         // Show error message
         $("#flashing-overlay-error").html(
-                '<div>' + err.message + '</div><div>' + 
-                config["translate"]["webusb"]["err"][errorMessage] +
-                '</div><a href="#" id="flashing-overlay-download">' +
+                '<div>' + ((err.message === undefined) ? "WebUSB Error" : err.message) + '<br >' + 
+                (err.name === 'device-disconnected' ? "" : config["translate"]["webusb"]["err"][errorMessage]) +
+                '<br ><a href="#" id="flashing-overlay-download">' +
```

Comment:
Device disconnected should not show the "Download Hex" link, only close (without the `|`):

![image](https://user-images.githubusercontent.com/29712657/63504156-d6566400-c4c8-11e9-9430-beafc686bd94.png)